src/components/common/YourCompany/EntityName.vue

```diff
@@ -347,20 +350,23 @@ export default class EntityName extends Mixins(CommonMixin, NameRequestMixin) {
    * - the business name has changed, and
    * - the filing is an alteration or firm change filing, and
    * - the name has not been changed by type
+   * - this is when a named business is changed to a numbered business
    */
```

Comment:
This method only returns true when changing a named business to numbered. So we can set the this.setNameChangedToNumber(result) here